editor/editor_help.h

```diff
@@ -198,6 +198,16 @@ class EditorHelp : public VBoxContainer {
 	static void _gen_extensions_docs();
 	static void _compute_doc_version_hash();
 
+	struct PropertyCompare {
+		_FORCE_INLINE_ bool operator()(const DocData::PropertyDoc &l, const DocData::PropertyDoc &r) const {
+			// Sort overridden properties above all else.
+			if (l.overridden == r.overridden) {
+				return l.name.naturalcasecmp_to(r.name) < 0;
+			}
+			return l.overridden > r.overridden;
```

Comment:
```suggestion
		_FORCE_INLINE_ bool operator()(const DocData::PropertyDoc &p_l, const DocData::PropertyDoc &p_r) const {
			// Sort overridden properties above all else.
			if (p_l.overridden == p_r.overridden) {
				return p_l.name.naturalcasecmp_to(p_r.name) < 0;
			}
			return p_l.overridden;
```
I'd say just checking the value is sufficient (and less confusing)

Reply:
So long as it achieves the same thing, sure, absolutely

Reply:
If both are `true` or `false` it enters the first part, otherwise it enters the second, and then it should be sorted first if `p_l` is overridden